Products/Products-Root/Screens/Authentication/Splash/SplashViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SplashViewController.swift
+//  Products
+//
+//  Created by Dan Pop on 10/03/2020.
+//  Copyright © 2020 Archlime. All rights reserved.
+//
+
+import UIKit
+
+protocol SplashViewControllerDelegate: class {
+    func splashViewControlledDidPressSignIn()
+    func splashViewControllerDidPressRegister()
+}
+
+
+class SplashViewController: BaseViewController {
+    private var viewModel: SplashViewModel!
+    weak var delegate: SplashViewControllerDelegate?
+    
+    
+    @IBAction func registerButtonPressed(_ sender: Any) {
```

Comment:
please move these two IBActions **after** the initializer method. Best practices say that this ordering must be preserved:


- IBOutlets first
- Variables
- View Cycle method overrides (viewDidLoad, viewWillAppear …)
- IBAction methods
- Public methods
- Private methods
- Extensions

Also please add MARKs